common/build.gradle

```diff
@@ -0,0 +1,6 @@
+apply plugin: "org.springframework.boot"
+apply plugin: 'io.spring.dependency-management'
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
spring boot starter web requires a main class

Reply:
Or spring in general

Reply:
Hehe thanks :)

Reply:
@qurben I've tried adding one, but it doesn't seem to resolve the issue, could you have a look?